Applications/Games/Overwatch/Online/script.js

```diff
@@ -19,10 +19,14 @@ var installerImplementation = {
             .category("Games")
             .executable("Overwatch.exe")
             .preInstall(function (wine/*, wizard*/) {
-                wine.windowsVersion("winxp");
+                wine.windowsVersion("win7");
                 wine.vcrun2015();
                 wine.corefonts();
                 wine.enableCSMT();
+                wine.DXVK(); // Needs approval, improves performance, DXVK 0.94 tested
+                // WINEDEBUG="-all" is mandatory for performance
+                wine.overrideDLL().set("builtin", ["d3d10", "d3d10_1", "d3d11", "dxgi", "d3d10core"]).do(); // Needs approval
```

Comment:
You can remove this, it is done inside the DXVK verb normally.

Reply:
based on my experience DLL Overrides for DXVK are by default set to prefer Native over buildin. https://youtu.be/Spr2jFWmhwU?t=1601 Notice the FPS in the game prior to setting DLL overrides. 

Had same issue on League Of Legends few days ago. 

Is `wine.DXVK();` correct syntax?

`WINEDEBUG="-all"` is also required and based on tests not present by default

Reply:
You're missing the import for `DXVK` at the beginning

Reply:
The problem with builtin is that it normally the does not use DXVK at all. You should do your test with `DXVK_HUD=1`.

Normally if you are launching the game from the library tab, WINEDEBUG is set to -all

Reply:
@ImperatorS79 Based on my test native results in game running on +- 6 FPS, Buildin was able to read +- 35 FPS. 

Noted WINEDEBUG, game was launched from terminal since the installation fails, but game is installed in wineprefix.

Reply:
@keyren I am nearly sure the game does not use DXVK. Use DXVK_HUD=1 env variable.

Reply:
@ImperatorS79 https://youtu.be/xPk8DllLLHQ

Possible? i'm confused since it outputs `WARNING: radv is not a conformant vulkan implementation, testing use only.` which to me indicates that DXVK is working?

testing

Reply:
OK, just set the dxvk version in the script then.

Reply:
For radv, it just means that the driver does not fully support vulkan.

Reply:
!!! WARNING !!! DXVK 0.93 causes exponensial regression once bot is shooted with a reaper's shotgut which bombs all cores on 100% and system needs to be restarted.

Reply:
Game runs on platinum using DXVK 0.91

Reply:
Wasn't able to reproduce the issue https://github.com/doitsujin/dxvk/issues/846

Recommends keeping DXVK (latest) and wait for feedback.